Apps/Src/main.c

```diff
@@ -4,9 +4,16 @@
  * Program for UTSVT BeVolt's Battery Protection System
  */
 
-#include "common.h"
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
```

Comment:
See my comments on `Apps/Inc/main.c`. Note that you will want to use this main.c and delete the other one

Reply:
use Apps/Inc/main.c and delete Apps/ Inc/ main.c